src/components/modules/escrow/ui/forms/InitializeEscrowForm.tsx

```
@@ -60,11 +67,25 @@ const InitializeEscrowForm = () => {
+        "roles.platformAddress",
+        selectedLoan.platformAddress || "",
+      );
+      form.setValue("roles.approver", selectedLoan.approver || "");
+      form.setValue("roles.releaseSigner", selectedLoan.releaseSigner || "");
+      form.setValue(
+        "roles.disputeResolver",
+        selectedLoan.disputeResolver || "",
+      );
+      form.setValue(
+        "milestones",
+        (selectedLoan.milestones || []).map((m: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)